reinforcement-learning/_posts/2018-12-17-tracto_td_learning.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: review
+title: "Tractography using reinforcement learning and adaptive-expanding graphs"
+tags: reinforcement-learning tractography
+author: "Antoine Théberge"
+cite:
+    authors: "Tingyi Wanyan, Lantao Liu, Eleftherios Garyfallidis"
+    title:   "Tractography using reinforcement learning and adaptive-expanding graphs"
+    venue:   "ISMRM 2015"
+pdf: "https://drive.google.com/open?id=1gYXIGYd6jRE0HCSvvx5NTRTBKUMLYBrb"
+---
+
+## Introduction
+
+Currently, three types of tractography algorithm exist: Deterministic, probabilistic and global algoritms. The authors propose a novel approach based on reinforcement-learning.
+
+## Method
+
+![](/reinforcement-learning/images/tracto-td-learning/tree.png)
+
+The authors used the ISMRM challenge dataset which provided a starting and goal point for tractography. They then used a graph to represent their exploration and experiences. Each node represent a state in their search and has a value function, updated via Temporal-Difference (TD) learning:
+
+$$ V(s_t) \leftarrow V(s_t) + \alpha[R_{t+1} + \gamma V(s_{t+1}) - V(s_t)] $$
+
+where $$V(s_t)$$ is the value function, the sum of all expected rewards at this node, $$\alpha$$ is the learning rate, $$\gamma$$ a discount factor and $$R_{t+1}$$ is the reward at the next time step. It is worth noting that the value function is only updated when a valid path (ed. the definition of a valid path is not specified, probably a path that goes near or to the end goal) is found to prevent penalizing invalid paths, which are still useful for exploration. In the end, optimal tracts are determined by their total reward.
```

Comment:
do they mention what their reward function is?

Reply:
Unfortunately not